### ISSUE TITLE
Big Outsider Spawn Problems

### ISSUE THREAD

**Issue by Delams-The-SM** [sojourn-13/sojourn-station#3955]
There is a rather **large** issue with how currently most Outsider spawns are mapped out, that being the cave spawns, preppers underground and even the swamp one.

Preppers spawn puts you in a small room with very little to get within it, that alone isn't a problem even if those robots can slowly whittle down your health with how many there can be but there is a much larger problem.

The very first airlock you open after having not enough resources to make yourself proper gear might contain a Forgotten Seeker if RNG hates you and unless it aggroes on some roaches you are Dead.

Something that has unfortunately happened to me multiple times, I managed to survive such spawn only once roundstart due to roaches that spawn in that first room having two Fuhrers that survived just long enough for me to get hits in on the damned thing and even then I got left bleeding, and if I were to spawn in any later into the round I most likely would be dead with no roaches to take the hits for me.

It really is annoying when literally you might not even step out of the room you spawn in to find yourself dead, and with the lack of tools, resources and space you can't really counter the Forgotten Seeker at all, granted once you do pass this first RNG spawn dice roll it is possible to survive one later on, with wood/metal/medicine and weapons you can gather at that point.

Forgotten Seeker being in spawn pool for this area does not make much sense, Outsiders at that point don't have enough resources/weapons/anything to deal with it and death is guaranteed unless in your RNG unluck you get lucky with RNG to have strong roaches fight alongside you and that is only for shift start, since after that no roach will be left alive to help you.

Simple fix would be to just . . . disable Forgotten Seeker spawning in the only room a freshly spawned surrounded by R walls outsider can go through to progress further.

On the same note the Swamps spawn has Mukwahs, the mutated bears that on a good day can dodge whole tube of slugs from a shotgun even but the actual problem with them is much more simple, random movement especially of other mobs that might attract their attention.

Especially for a late join that spawns there is a chance a Mukwah just happened to move away from the cave that is south east of the spawn, it is likely that a stray roach/spider/whatever attracts their attention outside while also possibly bringing out more than one.

North of the home also has a Mukwah for some reason, meaning that the player is stuck between Mukwahs and needs at least to fight one to go exploring further than their own house, that is if the mobs that spawn in and around the house don't attract Mukwah's attention directly to the house causing death most likely unless one is very lucky with weapons that spawn.

Granted unlike the preppers spawn this time the player is given resources, some space and weapons making being able to handle at least one Mukwah possible (unless that Mukwah literally side steps each slug you shot at them . . . something that did happen to me causing death just couple steps from where outsider spawns in)

But still, making the cave boarded up with wooden barricades would make their escapes outside less likely giving the outsider some space south of their spawn to explore in relative peace besides the roaches/spiders/traps that plague the swamps.

It would also fix the problem that the cave south east of the spawn is just a noob trap, if you do not know that Mukwahs lurk in the dark there you might want to take a peek in only to be greeted by a death welcoming committee:

![obraz](https://user-images.githubusercontent.com/66973657/190045331-c6aab280-52e7-45a8-951c-50203afca37b.png)

Do notice the dead roaches that caused the Mukwahs to bunch up and approach exit of the cave.

And just to note, Mukwahs are rather very deadly, just one hit from them can deal enough damage to cause slowdown large enough that even races with faster walk/run speed won't be quick enough to get away from them, they got alot of health on top of that and immunity to flashes, but there are things around the spawn that make them survivable, good firepower if you happen to roll a nice gun or a proper trap.

Both of those spawns suffer from possibility of a very bad RNG that causes high probability of Death if not straight up makes Death inevitable just couple steps from where you spawn in especially preppers that really does not need a Forgotten Seeker to spawn in outside of the room player wakes up in and swamps could just have the welcoming committee of death boarded up, giving the player at least a moment to set up a trap or even run away from mobs they might not be equipped to handle so early into their round.

That RNG especially on prepper spawn can make you die less than 3 minutes in causing you to obviously not play the same character for the next 5-6 hours, granted unfair RNG is not the only problem with outsider spawns.

The caves spawns suffer from the fact player needs to know prior information if they want to get out, and I don't mean knowing where ladders are because being lost within caves can be fun, exploring them and such but while most mobs down there are manageable with few exceptions that need more proper equipment there is one mob that is a guaranteed Death.

The Nightmare Stalker has enough health, damage and speed to outrun, outdamage and outlive whatever a freshly spawn outsider could throw at them and with the fact they are 'invisible' one will notice them most likely too late to run away.

Be it shields, home-wrecker, starting weapons even things one might find within the caves, there is nothing that will save a outsider from that creature causing cave exploration to simply become "Memorise routes towards ladder from each spawn or risk death less than 30 minutes in"

Now granted with just 3 spawns affected by it one could potentially memorise where those things are, granted unless someone spends some time observing the round to find out where those things are that means at least 3 rather unfair feeling deaths + ones caused from forgetting a path if you happened to not get specific spawn for a while.

The reason those things exist I think are to protect c4 that spawn underground as loot ? then why not just . . . move them to a single path with c4 at the end of it with the path starting on a "DANGER EXPLOSIVES" sign and a wooden barrier ? it would make the danger obvious and when someone goes in they will learn a bitter lesson just once rather than relearing it from time to time with nightmares just being out in the open within one of the passageways.

### **TLDR**

The **Preppers** spawn can spawn a Forgotten Seeker outside of the room you spawn in making surviving impossible.
**Swamps** have Mukwahs around where player spawns that with unlucky RNG of other mobs can get closer to where player spawns in a horde even making survival impossible.
**Caves** have Nightmare Stalkers that _require_ knowledge of where they spawn if player wants to exit the caves since they are certain Death.

Those three things could really use improvement since fixing them would reduce number of unfair feeling deaths at the very start of the round and they can be oddly common if RNG happens to hate you, I happen to be on 4 round streak with those things.

**Comment by Spoopy11** [comment]
Outsider is hard-mode.  A large part skill with a small touch of skill.  The only one thing I agree with is the seekers bit, but I don't want to touch upon that since it involves trying to balance Preppers entirely (Since someone added seekers to the Preppers spawn pool)

In general Preppers could do with being largely changed to suit new difficulties and make for better rewards.

**Comment by Trilbyspaceclone** [comment]
You should be able to deal with a seeker and head to the chem dispender for healing juices.
I fail to see the issue

If you have 2 seeker spawns then you might legitmetnly be unable to tho

**Comment by Trilbyspaceclone** [comment]
If proper issue is found please make a new issue report otherwise I see no error in code at this moment
